docs/quickstart.md

```diff
@@ -244,16 +244,16 @@ Let test **SPIKE** by creating a secret
 spike init
 
 # Register a secret:
-spike put /secrets/db-secret username=postgres password=postgres
+spike secret put /secrets/db-secret username=postgres password=postgres
```

Comment:
`/secrets/db/secret` is the corerct syntax as it suggests that a path to a secret can be like a unix path.

Similar to `/tenants/pepsicola/db/creds`

The current `db-secret` loses tha meaning, so I suggest changing it back to something like `/secrets/db/secret` or maybe something more examplary like `/tenants/acme/db/creds`.